ui/src/core_components/input/CoreFileInput.vue

```diff
@@ -105,30 +133,37 @@ const encodeFile = async (file: File) => {
 
 const fileChange = async (ev: InputEvent) => {
 	const el = ev.target as HTMLInputElement;
-	if (!el.files || el.files.length == 0) return;
+	handlefiles(el.files);
+};
+
+const handlefiles = async (Files: FileList) => {
+	if (!Files || Files.length == 0) return;
+	window.alert("handlefiles");
```

Comment:
I got this alert message when I uploaded the file. Is this a development artifact that remained in the code?